nft/handler.go

```diff
@@ -59,8 +59,7 @@ func (g grpcHandler) MintNFT(ctx context.Context, request *nftpb.NFTMintRequest)
 	}
 
 	return &nftpb.NFTMintResponse{
-		TokenId: resp.TokenID,
-		Header:  &nftpb.ResponseHeader{TransactionId: resp.TransactionID},
+		Header: &nftpb.ResponseHeader{TransactionId: resp.TransactionID},
```

Comment:
Cool, cleaner. Lets make sure that the functional tests are updated if needed :)